res/database/Default/user/scr0llbaer.xml

```diff
@@ -467,30 +510,121 @@ Format: Filmed lectures. Laid-back and to lay down.</en>
 				<modifier name="topicality::age" value="5" />
 				<modifier name="topicality::refresh" value="2" />
 			</programme_data_modifiers>
-		</scripttemplate-->
+		</scripttemplate>
 
-		<!--scripttemplate product="3" licence_type="1" guid="9f1cacc4-aa56-4f4a-9a9f-8c21c4bd3a7d" creator="9551" comment="Perhaps some reciprocal effect with the corresponding news genre can be implemented? TODO: I am not happy with the titles and descriptions of these.">
+		<scripttemplate product="3" licence_type="1" guid="56bcb1ff-8a5d-4d91-b3f7-ad6958b3855b" creator="9551">
+			<title>
+				<de>%recreational%%athletes% aus %randomcity% und anderswo</de>
+				<en>%recreational% %athletes% from %randomcity% and elsewhere</en>
+			</title>
+			<description>
+				<de>Aktuelle News und Berichterstattungen aus den Bunt-, Amateur- und Kreisligen der Nation. Fußball, Eishockey, Basketball, Football, Tennis, Tischtennis, Leichtathletik, Kirmesboxen, Kartsport und Formel 3 &amp; 4, und vieles mehr.</de>
```

Comment:
We could make it less serious - not just the Kirmesboxen. Guess after 3-4 popular sports we could move on to
"Bierpong", "Trocken-Curling", "Nagel-Weitklipsen" ... 

... take with a grain of salt, just a suggestion ;)

Reply:
That would get too long then I think, and I was intending to keep it to existing sports ("Kirmesboxen" actually used to be quite serious business, they made a living of it).

We could incorporate those in some kind of "[Spiel ohne Grenzen](https://de.wikipedia.org/wiki/Spiel_ohne_Grenzen)" format another time.

Reply:
Added TODO